app/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
     alias(libs.plugins.devtools)
     alias(libs.plugins.about.libraries)
     id("kotlin-parcelize")
-    kotlin("plugin.serialization") version "1.9.0"
+    kotlin("plugin.serialization") version "1.9.10"
```

Comment:
The previous `kotlinx.serialization` bump caused a crash at runtime. Can you get past the auto updates dialog with both options selected without crashing?

Reply:
Yes that works, at least for me on Pixel 4 device with Android 13.